IREC/IREC_PDD.tex

```diff
@@ -0,0 +1,383 @@
+
+\subsection{Payload Details}
+\label{subsec:Payload Details}
+Hyperion is modeled after the black box technology found in aircraft. It represents a combination of two initially separate concepts of extreme self-diagnostics
+and an airbag deployment mechanism for assisted landing similar to the Spirit and Opportunity Mars rovers. As per the IREC rules and requirements, the payload will
+follow a traditional CubeSat standard. It is estimated that a 3U (10cm x 10cm x 30cm) volume will be utilized.
+
+On the self-diagnostics side, data gathered from the rocket and the payload will be written to nonvolatile
+memory as well as transmitted to a receiving ground station. The data gathered within the payload includes linear acceleration, angular rate, magnetic field detection,
+temperature, absolute barometric pressure, GPS information, pressure within the landing balloons, and a variety of analog measurements from the circuit itself. Other data
+gathered from the rocket includes vibration, mechanical strain, and a potential for engine data depending on the engine selected for use by LI.
+
+During apogee at 10,000 feet, Hyperion will separate from the rocket and fall back to Earth with a controlled descent from a parachute. At a sufficient altitude, Hyperion will
+electrically open valves that control the flow of CO2 from compressed canisters to inflate the balloons before striking the ground with a designed energy. These designed impact
+conditions will allow for proof of concept without subjecting the payload to unnecessary stresses.
+
+\subsection{Timeline}
+\label{subsec:Timeline}
+  % Be as detailed as you can, but it's okay if there are unknowns.
+  % At the very least, specify how many semester you expect the project to take until it reaches completion.
+The timeline is based largely around two important dates. The first is the hand-off of the external mechanical design to LI on November 1st which will provide time for LI
+to incorporate Hyperion into their own designs. The second important date is the competition itself. Although the specific day has not been announced yet, past competition dates
+suggest that it will be held during late June. The timeline is as follows:
+\begin{itemize}
+  \item September 1st: Begin narrowing down payload ideas using a decision matrix based on the SDL Payload Challenge scoring rubric
+  \item September 8th: Decide on the specific payload and make a plan of action for research and design
+  \item October 1st: Present a preliminary design with estimates for size, cost, and weight
+  \item November 1st: Mechanical design delivery to LI. No changes to the external frame can be made after this point
+  \item Semester's End: Final hardware design completed and materials ordered
+  \item April 1st: Integration testing of rocket and payload
+  \item April/May: Test launch of rocket and payload
+  \item June 22nd-24th: Intercollegiate Rocket Engineering Competition
+\end{itemize}
```

Comment:
I think relative timelines belong in a PDD for reasons mentioned above. Also, tables > bullets